### ISSUE TITLE
DocX => pdf styles missing

### ISSUE THREAD

**Issue by robov** [PHPOffice/PHPWord#1139]
When I convert my docx at least the following styles are missing
* headers
* footers
* bullets
* bold
* custom style definitions in work (ie I had a TITLE style.. big center... but this was rendered in pdf as a normal style

**Comment by robov** [comment]
And the styles are not there when I use TCPDF and not when I use DOMPDF
Both render the pdf in the same style-less matter

**Comment by robov** [comment]
When I save the original file as a word97 file..the styling is even worse...
```
        $filename = "interview97.doc";
        $source = $path . $filename;
        $phpWord = \PhpOffice\PhpWord\IOFactory::load($source,  'MsDoc'); 
        
        $objWriter = \PhpOffice\PhpWord\IOFactory::createWriter($phpWord, 'PDF');
        $objWriter->save($path . "interview97.pdf" );  
```

**Comment by FBnil** [comment]
What I am using is OpenOffice (`yum install libreoffice-writer`)

```
$cmd = "export HOME=/tmp && /bin/oowriter --headless --convert-to pdf --outdir '" . $path . "' '" . $fqn . "' 2>&1" ;
$output = shell_exec($cmd);
```
The installation size is enormous, but the results are quite good.  :( Sorry PHPWord!

**Comment by robov** [comment]
Yes I switched to...as the results are horrible in phpword and great in libre

**Comment by Sarke** [comment]
This seems related to the HTML issue: #1047 

The PDF writers being used are taking in the HTML output, which also lacks the styling.  The classes are being defined in the `<style>` tag, but they are just not being used.

**Comment by chrisribe** [comment]
This still seems to be an issue. html and pdf outputs do not replicate the some styles in docx (header / footers).

**Comment by SenFire** [comment]
Issue still present on 0.16.0

**Comment by TheKeymaster** [comment]
If you have libreoffice installed you can use:

```php
exec('libreoffice --headless --convert-to pdf path/to/some.docx path/to')
```

I wouldn't say it is a good solution but it is deffinetely a reliable one.

**Comment by alexmanotas** [comment]
can someone tell me please, how I can install libreoffice on my server or how to include it in my files to call it in my php file.

**Comment by downwater** [comment]
For information, a [question in StackOverflow](https://stackoverflow.com/a/54660038/4375327) about this issue.

**Comment by xeemoon** [comment]
I have the same that Issue, Kindly helps me, someone. Thanks, @alexmanotas can you do it?
